manager/src/main/java/com/redhat/service/smartevents/manager/models/Bridge.java

```diff
@@ -71,6 +73,9 @@ public class Bridge extends ManagedDefinedResource<BridgeDefinition> {
     @Column(name = "region", nullable = false, updatable = false)
     private String region;
 
+    @Column(name = "subscription_id", nullable = false, updatable = false)
```

Comment:
@r00ta Minor: We're declaring `nullable=false` here, but the schema currently allows null values.